More/EAV/Attribute.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace DesignPatterns\More\EAV;
+
+/**
+ * Class Attribute
+ */
+class Attribute implements ValueAccessInterface
+{
+    /**
+     * @var array|Value[]|ValueInterface[]
+     */
+    private $values = array();
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * @return array|Value[]|ValueInterface[]
+     */
+    public function getValues()
+    {
+        return $this->values;
+    }
+
+    /**
+     * @param Value|ValueInterface $value
+     * @return $this
```

Comment:
I still insist on `@return $this`. The IDEs understand it nice. And it useful for inheritance when extends `Attribute` class with other one.